### ISSUE TITLE
Cost-Oriented Load Forecasting. (arXiv:2107.01861v1 [eess.SY])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#8621]
Cost-Oriented Load Forecasting. (arXiv:2107.01861v1 [eess.SY])<br>
https://ift.tt/3jRGOoV<br>
<p>Accurate load prediction is an effective way to reduce power system operation costs. Traditionally, the mean square error (MSE) is a common-used loss function to guide the training of an accurate load forecasting model. However, the MSE loss function is unable to precisely reflect the real costs associated with forecasting errors because the cost caused by forecasting errors in the real power system is probably neither symmetric nor quadratic. To tackle this issue, this paper proposes a generalized cost-oriented load forecasting framework. Specifically, how to obtain a differentiable loss function that reflects real cost and how to integrate the loss function with regression models are studied. The economy and effectiveness of the proposed load forecasting method are verified by the case studies of an optimal dispatch problem that is built on the IEEE 30-bus system and the open load dataset from the Global Energy Forecasting Competition 2012 (GEFCom2012).</p>
<br>
<br>
via eess.SY updates on arXiv.org http://arxiv.org/<br>
